### PR TITLE
feat(wallet-connect): support deeplinks

### LINKED ISSUE #3220: Deeplinks are not working with Uniswap DEX
**Describe the bug**
I tried to swap some ETH to WETH on Uniswap and after I click on swap button it doesn't open SH wallet confirmation popup or redirect me to SH wallet, I have to open wallet manually. Idk if it's expected behaviour, it looks like deeplinks are not working and app behaves like web wallet

**To Reproduce**
1. Go to Uniswap and connect the wallet via wallet connect 
2. Try to swap ETH to WETH
3. See error

**Expected behavior**
When I do swap on DEX I have to be redirected to wallet to sign confirmation modal 

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Samsung S9
 - Browser chrome

**Additional context**
reproducible on dev branch 

https://github.com/user-attachments/assets/41663bcb-0473-4f3f-9ad5-42333a5c21f6

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixes #3220

### CODE REVIEW

File: src/composables/walletConnect.ts [hunk 1]
```diff
@@ -74,6 +81,13 @@ export function useWalletConnect({ offscreen } = { offscreen: false }) {
 
   const ethAccounts = computed(() => accountsGroupedByProtocol.value[PROTOCOLS.ethereum] || []);
 
+  function cloaseAppIfOpenUsingDeeplink() {
```
Comment (martinkaintas) [inline src/composables/walletConnect.ts:84]:
```suggestion
  function closeAppIfOpenUsingDeeplink() {
```

**Review by martinkaintas — CHANGES_REQUESTED** [review]

**Review by martinkaintas — APPROVED** [review]

**Comment by Liubov-crypto** [comment]
This seems not to be fixed. I have to open app manually to confirm the Tx: 


https://github.com/user-attachments/assets/a34b4a75-d2c3-41ac-bf26-b0cbd449df05

**Comment by CedrikNikita** [comment]
> This seems not to be fixed. I have to open app manually to confirm the Tx:

New logic would only work if user was initially redirected to the wallet via deeplink.

**Comment by Liubov-crypto** [comment]
retested, LGTM

**Pull request merged**